distributor-api-v1/use-cases/on-session-payments.md

```diff
@@ -0,0 +1,30 @@
+# On session payments
+
+Whenever a user create reservation there is an option to allow him pay directly. This needs to be correctly configured via
+settlement rules on `RateGroup`. We provide this functionality only for `Rates` where the payment should be automatically
+taken immediately when the reservation group is created. In order to simplify the workflow we offer this functionality
+in our payment gateway via a feature called "Payment request".
+
+## Creating a reservation group
+
+First step of the entire workflow is to [create a reservation group via our API](../operations.md#request-apibaseurlapidistributorv1reservationgroupscreate).
```

Comment:
Maybe this link would be better - https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/operations#create-reservation-group ?